tests/Backend/CommonPart1/BucketsTest.php

```diff
@@ -232,6 +232,8 @@ public function testBucketsListWithIncludeMetadata(string $devBranchType, string
         $buckets = $this->_testClient->listBuckets([
             'include' => 'metadata',
         ]);
+        $this->assertNotEmpty($buckets, 'There should be at least one bucket prepared for the testcase');
```

Comment:
IMHO to má být pod tím. Furt tam můžeš mít buckety, jen nebudou mít správné idčko a filtry vyjde prázdný, ne?

Reply:
omg, jasne... sorry